### ISSUE TITLE
Getting NullReferenceException from Sitelet.InferWithCustomErrors

### ISSUE THREAD

**Issue by lierdakil** [dotnet-websharper/core#902]
Here's a relevant portion of the backtrace:
```
[NullReferenceException: object reference not set to an instance of an object.]
   WebSharper.Sitelets.ServerInferredOperators.IWithCustomErrors(Type typ, InferredRouter item) +164
   WebSharper.Sitelets.Router.IInferWithCustomErrors() +57
   WebSharper.Sitelets.SiteletModule.InferWithCustomErrors(FSharpFunc`2 handle) +81
```

Let me know if I can help debug this.

**Comment by Jand42** [comment]
Thanks for the report!

As a quick workaround, try `Sitelet.Infer<ParseRequestResult<T>>` instead, where `T` is your endpoint type.
This should be equivalent, only there is a bug currently in `InferWithCustomErrors`.

**Comment by Jand42** [comment]
Ok, sorry, not entirely equivalent, because `InferWithCustomErrors` also fixes up the `Context` object to create links for type `T` instead of `ParseRequestResult<T>`.

You can do this too with `let ctx = (Context.Map ParseRequestResult.Success ctx)`.

**Comment by lierdakil** [comment]
Thanks for quick response and for suggesting a workaround!

**Comment by Jand42** [comment]
@lierdakil Fix is now released in [4.1.6.207](https://github.com/dotnet-websharper/core/releases/tag/4.1.6.207).
